docs/settings/security-settings.asciidoc

```diff
@@ -239,3 +239,113 @@ The format is a string of `<count>[ms\|s\|m\|h\|d\|w\|M\|Y]` (e.g. '20m', '24h',
+| Set to `true` to enable audit logging for security events. *Default:* `false`
+|===
+
+[float]
+[[audit-logging-settings-beta]]
+===== ECS audit logging settings
+
+beta[]
+
+To enable the <<xpack-security-ecs-audit-logging, ECS audit logger>>, specify where you want to write the audit events using `xpack.security.audit.appender`.
+
+[cols="2*<"]
+|===
+| `xpack.security.audit.appender`
+| experimental[] Optional. Specifies where audit logs should be written to and how they should be formatted.
+
+2+a| For example:
+
+[source,yaml]
+----------------------------------------
+xpack.security.audit.appender:
+  kind: file
+  path: /path/to/audit.log
+  layout:
+    kind: json
+----------------------------------------
+
+| `xpack.security.audit.appender.kind`
+| experimental[] Required. Specifies where audit logs should be written to. Allowed values are `console` or `file`.
+|===
+
+[float]
+[[audit-logging-file-appender]]
+===== File appender
+
+The file appender can be configured using the following settings:
+
+[cols="2*<"]
+|===
+| `xpack.security.audit.appender.path`
+| experimental[] Required. Full file path the log file should be written to.
+
+| `xpack.security.audit.appender.layout.kind`
+| experimental[] Required. Specifies how audit logs should be formatted. Allowed values are `json` or `pattern`.
+|===
+
+[float]
+[[audit-logging-pattern-layout]]
+===== Pattern layout
+
+The pattern layout can be configured using the following settings:
+
+[cols="2*<"]
+|===
+| `xpack.security.audit.appender.layout.highlight`
+| experimental[] Optional. Set to `true` to enable highlighting log messages with colors.
+
+| `xpack.security.audit.appender.layout.pattern`
+| experimental[] Optional. Specifies how the log line should be formatted. *Default:* `[%date][%level][%logger]%meta %message`
+|===
+
+[float]
+[[audit-logging-ignore-filters]]
+===== Ignore filters
+
+[cols="2*<"]
+|===
+| `xpack.security.audit.ignore_filters[]`
+| experimental[] List of filters that determine which events should be excluded from the audit log. An event will get filtered out if at least one of the provided filters matches.
+
+2+a| For example:
+
+[source,yaml]
+----------------------------------------
+xpack.security.audit.ignore_filters:
+- actions: [http_request] <1>
+- categories: [database]
+  types: [creation, change, deletion] <2>
+----------------------------------------
+<1> Filters out HTTP request events
+<2> Filters out any data write events
+
+| `xpack.security.audit.ignore_filters[].actions[]`
+| experimental[] List of values matched against the `event.action` field of an audit event. Refer to <<xpack-security-audit-logging>> for a list of available events.
+
+| `xpack.security.audit.ignore_filters[].categories[]`
+| experimental[] List of values matched against the `event.category` field of an audit event. Refer to https://www.elastic.co/guide/en/ecs/current/ecs-allowed-values-event-category.html[ECS categorization field] for allowed values.
```

Comment:
Should we link to the ECS version that we're claiming to support instead of `current`?

```suggestion
| experimental[] List of values matched against the `event.category` field of an audit event. Refer to https://www.elastic.co/guide/en/ecs/1.5/ecs-allowed-values-event-category.html[ECS categorization field] for allowed values.
```